source/connect/required-access.txt

```diff
@@ -0,0 +1,97 @@
+.. _required-access:
+
+===============
+Required Access
+===============
+
+.. default-domain:: mongodb
+
+With authentication/authorization enforced on the connected MongoDB
```

Comment:
Perhaps:

"Compass users require specific privileges to access various Compass features if authentication/authorization is enforced on the connected MongoDB instance."

Reply:
I'd also make each sentence its own paragraph.